ecms_base/ecms_base.install

```diff
@@ -895,3 +895,34 @@ function ecms_base_update_9054(array &$sandbox): void {
   }
 
 }
+
+/**
+ * Updates to run for the 0.5.5 tag.
+ */
+function ecms_base_update_9055(array &$sandbox): void {
+
+  // Install the new module.
+  $modules_to_install = [
+    'paragraphs_translation_sync',
+  ];
+
+  \Drupal::service('module_installer')->install($modules_to_install);
+
+  // Config updates.
+  $path = \Drupal::service('extension.list.profile')->getPath('ecms_base');
+
+  /** @var \Drupal\Core\Config\StorageInterface $active_storage */
+  $active_storage = \Drupal::service('config.storage');
+
+  /** @var \Drupal\Core\Config\FileStorage $install_source */
+  $install_source = new FileStorage($path . "/config/install/");
+
+  $updatedConfig = [
+    'user.role.anonymous',
```

Comment:
Same kinda question as the other call out.

Reply:
@JMStewart00  Yea, it's an odd work-around but it seemed to solve the issue noted here:
https://www.drupal.org/project/paragraphs/issues/2902371#comment-12577532

I tested and anonymous can't access any edit screens

Reply:
oh cool.